main/src/cgeo/geocaching/connector/oc/OkapiClient.java

```diff
@@ -309,6 +309,38 @@ public static LogResult postLog(final Geocache cache, final LogType logType, fin
         return new LogResult(StatusCode.LOG_POST_ERROR, "");
     }
 
+    public static boolean UploadPersonalNotes(final OCApiConnector connector, final Geocache cache) {
+        Log.d("Uploading personal note for opencaching");
+
+        final Parameters notesparam = new Parameters("cache_code", cache.getGeocode(), "fields", getFullFields(connector));
```

Comment:
Reading this line again, I would say you do not need to request all fields just to retrieve the old value of the note, the personal note should suffice.